File: src/1672-Richest-Customer-Wealth/richest_customer_wealth.py
from typing import List

class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        max_wealth: int = 0
        for customer in accounts:
            customer_wealth: int = 0
            for bank in customer:
                customer_wealth += bank
            max_wealth = max(max_wealth, customer_wealth)
        return max_wealth


if __name__ == "__main__":
    accounts : List[List[int]] = [[1,2,3],[3,2,1]]
    solution = Solution()
    print(solution.maximumWealth(accounts ))

File: src/1672-Richest-Customer-Wealth/richest_customer_wealth.cpp
#include <iostream>
#include <vector>

class Solution {
public:
    int maximumWealth(std::vector<std::vector<int>>& accounts) {
        int max_wealth = 0;
        for (auto customer: accounts) {
            int customer_wealth = 0;
            for (auto bank: customer) {
                customer_wealth += bank;
            }
            max_wealth = std::max(max_wealth, customer_wealth);
        }
        return max_wealth;
    }
};

int main() {
    std::vector<std::vector<int>> accounts {{1,2,3}, {3,2,1}};
    Solution solution;
    int result = solution.maximumWealth(accounts);

    std::cout << result;

    return 0;
}